include/alpaka/queue/QueueGenericSyclBlocking.hpp

```diff
@@ -16,10 +16,10 @@
 #    include <memory>
 #    include <utility>
 
-namespace alpaka::experimental
+namespace alpaka
 {
     template<typename TDev>
-    using QueueGenericSyclBlocking = detail::QueueGenericSyclBase<TDev, true>;
-} // namespace alpaka::experimental
+    using QueueGenericSyclBlocking = trait::QueueGenericSyclBase<TDev, true>;
+} // namespace alpaka
```

Comment:
That namespace change feels also wrong. `QueueGenericSyclBase` is an implementation detail, not a trait/customization point.

Reply:
Reverted that change as well. Now `QueueGenericSyclBlocking` and `QueueGenericSyclBase` are both in `namespace alpaka` while the underlying implementation (`QueueGenericSyclImpl` is in `alpaka::detail`)